src/components/Pages/Unit/widgets/UnitMaterials/Transcription/Transcription.js

```diff
@@ -139,9 +162,21 @@ class Transcription extends Component {
     this.setState({ selectedFile, language: newLanguage });
   };
 
+  prepareScrollToSearch = (data, search) => {
+    const result = data.split('<p').map(p => {
+      const clearTags = p.replace(/<.+?>/gi, '');
```

Comment:
p.replace(/<.+?>/gi, '');

This will delete the </p> closing tags.

See my document an idea how to solve this.